front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -206,43 +211,40 @@ export default function AssistantBuilder({
   };
 
   const formValidation = useCallback(async () => {
-    let valid = true;
+    const modelConfig = SUPPORTED_MODEL_CONFIGS.filter(
+      (config) =>
+        config.modelId === builderState.generationSettings.modelSettings.modelId
+    )[0];
+    if (!modelConfig) {
+      // unreachable
+      throw new Error("Model configuration not found");
+    }
 
-    const { handleValid, handleErrorMessage } = await validateHandle({
+    const { handleErrorMessage } = await validateHandle({
       owner,
       handle: builderState.handle,
       initialHandle: initialBuilderState?.handle,
       checkUsernameTimeout,
     });
-    valid = handleValid;
-    setAssistantHandleError(handleErrorMessage);
+    const localHandleError = handleErrorMessage;
```

Comment:
Why going through a second var for this? 

You could do 
```js
const { handleErrorMessage: localHandleError } = await validateHandle({
      owner,
      handle: builderState.handle,
      initialHandle: initialBuilderState?.handle,
      checkUsernameTimeout,
    });
    setAssistantHandleError(localHandleError);
```

or simply : 
```js
const { handleErrorMessage } = await validateHandle({
      owner,
      handle: builderState.handle,
      initialHandle: initialBuilderState?.handle,
      checkUsernameTimeout,
    });
    setAssistantHandleError(handleErrorMessage);
```

Reply:
because i'm stupid. fixing :)